sample.env

```diff
@@ -45,6 +45,6 @@ BACKEND_STRIPE_ITEM_SINGLE_TO_FAMILY=
 
 # URL ending without / because backend adds queries
 # URLS to where the user will be redirected in the frontend after the checkout process is completed
-BACKEND_STRIPE_SUCCESS_URL=http://localhost:3000/last-will/order-confirmation
-BACKEND_STRIPE_CANCEL_URL=http://localhost:3000/last-will/order-confirmation
+BACKEND_STRIPE_SUCCESS_URL=http://localhost:3000/last-will/order-confirmation?success=true
+BACKEND_STRIPE_CANCEL_URL=http://localhost:3000/last-will/order-confirmation?success=false
```

Comment:
this changed entirely, pls do not edit this in frontend PRs, this is also really bad, because we add the queries manually upon creating the checkout session, see #305 for more information